packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -339,11 +339,6 @@ export default function CollabSidebar() {
 		? AddCommentToolbarButton
 		: AddCommentButton;
 
-	// If postId is not a valid number, do not render the comment sidebar.
-	if ( ! ( !! postId && typeof postId === 'number' ) ) {
-		return null;
-	}
```

Comment:
I'm wondering whether we should keep this code just to be safe.

Reply:
I'm guessing eventually, the editor will support Block Comments for templates. It could be handy when collaborating on design.

That would mean selectors need to adapt to them, rather than just using early return.

Reply:
Should we keep this for now?

Reply:
Yes, let's keep this just in case. Updated in ee85fcdab2fb14b8a7d6918db6cfae26a850c8a2